crates/biome_js_formatter/tests/specs/ts/arrow/issue-2736.tsx

```diff
@@ -0,0 +1,19 @@
+type MenuDividerProps = {};
```

Comment:
Would you move the test to the `tests/specs/tsx` directory if it's a `ts` file? Otherwise, it won't be executed.https://github.com/biomejs/biome/blob/0a3588efc49936633433426e76bc7bc684735c0c/crates/biome_js_formatter/tests/spec_tests.rs#L14-L16

https://github.com/biomejs/biome/blob/0a3588efc49936633433426e76bc7bc684735c0c/crates/biome_js_formatter/tests/spec_tests.rs#L21-L23

Reply:
Whoops, my bad. Should be good now.

Reply:
No worries